packages/@uppy/companion/src/server/provider/zoom/adapter.js

```diff
@@ -31,7 +31,7 @@ exports.getDateName = (start, end) => {
 }
 
 exports.getAccountCreationDate = (results) => {
-  return moment(results.created_at)
+  return moment.utc(results.created_at)
```

Comment:
zoom api timestamps are in UTC but it looks like `moment()` uses the local server time by default. this fixes that - see https://momentjs.com/docs/#/parsing/